### DISCUSSION TITLE
Landscape City Blurred [3.5"]

### DISCUSSION THREAD

**Discussion by Higo709** [mathoudebine/turing-smart-screen-python#602] [Themes]
A custom version of the Landscape6Grid theme.
It shows, in order from left to right, CPU usage, CPU temp, Disk space used, GPU usage, GPU temp and RAM Usage.

Hope you like it!

![LandscapeCityBlurredPreview](https://github.com/user-attachments/assets/da1713d0-8957-4851-9fcc-1572f5b3f532)

[LandscapeCityBlurred.zip](https://github.com/user-attachments/files/17705674/LandscapeCityBlurred.zip)

**Comment by DJRoby19** [comment]
But it's not actually in Yaml format..for turingsmartscreen

**Reply by Higo709** [reply]
I uploaded the wrong file 😅 fixed! thank you for your feedback!

**Comment by kawaiinogoth** [comment]
Наконец, что-то годное
